src/server/src/main/java/io/cassandrareaper/storage/IDistributedStorage.java

```diff
@@ -32,12 +32,8 @@
 
   boolean takeLead(UUID leaderId);
 
-  boolean takeLead(UUID leaderId, int ttl);
-
   boolean renewLead(UUID leaderId);
 
-  boolean renewLead(UUID leaderId, int ttl);
-
```

Comment:
i took this stuff out because it's not used yet. if needed it should be included later in the side-car PR.

Reply:
I pushed a commit to address this.